pyproject.toml

```diff
@@ -2,6 +2,83 @@
 requires = ['setuptools', 'wheel', 'requests']
 build-backend = 'setuptools.build_meta'
 
+[project]
+name = "rohmu"
+authors = [
+  { name="Aiven", email="opensource@aiven.io" },
+]
+description = "Rohmu is a python library providing an interface to various cloud storage providers."
+readme = "README.rst"
+requires-python = ">=3.8"
+license = {text = "Apache-2.0 license"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Database :: Database Engines/Servers",
+    "Topic :: Software Development :: Libraries :: Python Modules"
+]
+dependencies = [
+    "azure-common >= 1.1.28, < 2",
+    "azure-storage-blob >= 2.1.0",
+    "botocore",
+    "cryptography",
+    "google-api-python-client",
+    "httplib2",
+    "oauth2client",
+    "paramiko",
+    "pydantic < 2",
+    "python-dateutil",
+    "python-swiftclient",
+    "requests",
+    # Kept in sync with Fedora versions
+    "python-snappy==0.6.1",
+    "zstandard==0.21.0",
```

Comment:
I don't think it should pin versions here in the non-dev dependencies.
(The pinned versions where in the requirements.dev.txt and not in the setup.cfg).

Reply:
Reverting the "silent" change to be strictly compatible with the previous format.

To be fair, I don't think the "dev" group is the right place to pin these dependencies, they would be better suited to a "compat" group. Out of scope for this PR nonetheless.